src/System/Wlog/LoggerConfig.hs

```diff
@@ -351,3 +354,24 @@ maybeLogsDirB prefix = mempty { _lcLogsDirectory = prefix }
 -- | Setup 'lcLogsDirectory' inside 'LoggerConfig' to specific prefix.
 logsDirB :: FilePath -> LoggerConfig
 logsDirB = maybeLogsDirB . Just
+
+-- | Lens to help to change some particular logger properties.
+--
+-- For example if you want to use default configurations,
+-- but need to change logger's severity to 'warningPlus'
+-- you can do it this way:
+--
+-- @
+-- 'System.Wlog.Launcher.launchWithConfig'
+--     ( defaultConfig "myLogger"
+--     & 'atLogger' "myLogger"
+--     . 'ltSeverity' ?~ 'warningPlus' )
+--     "myLogger"
+-- @
+--
+atLogger :: LoggerName -> Traversal' LoggerConfig LoggerTree
+atLogger logName = lcTree . leveldown (LoggerName <$> Text.splitOn "." (getLoggerName logName))
+  where
+    leveldown []     = error "Logger should be provided"
```

Comment:
Could you please `logName` to error `message`? Btw, here you should use `bug` from `universum` instead of `error`.

Another possible solution is to return `Nothing` when there's no such logger name. 

Which one do you prefer more, @vrom911?